src/validations/test/rules/sar.xspec

```diff
@@ -2,35 +2,305 @@
+
     <x:scenario
         label="In FedRAMP OSCAL SAR">
+
         <x:scenario
-            label="when the metadata element">
-            <x:scenario
-                label="is present">
-                <x:context>
-                    <assessment-results
-                        xmlns="http://csrc.nist.gov/ns/oscal/1.0">
-                        <metadata />
-                    </assessment-results>
-                </x:context>
-                <x:expect-not-assert
-                    id="metadata-element-exists"
-                    label="that is correct" />
-            </x:scenario>
-            <x:scenario
-                label="is absent">
-                <x:context>
-                    <assessment-results
-                        xmlns="http://csrc.nist.gov/ns/oscal/1.0" />
-                </x:context>
-                <x:expect-assert
-                    id="metadata-element-exists"
-                    label="that is incorrect" />
+            label="import-ap">
+
+            <x:context>
+                <assessment-plan
+                    xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                    <import-ap
+                        href="#bf974aa0-d662-4aef-a7a5-8f4cce398746" />
+                    <back-matter>
+                        <resource
+                            uuid="bf974aa0-d662-4aef-a7a5-8f4cce398746">
+                            <title>Security Assessment Plan</title>
+                            <prop
+                                name="type"
+                                value="security-assessment-plan" />
+                            <rlink
+                                href="../../../../dist/content/templates/ssp/xml/FedRAMP-SAR-OSCAL-Template.xml"
+                                media-type="text/xml" />
+                        </resource>
+                    </back-matter>
+                </assessment-plan>
+            </x:context>
+
+            <x:scenario
+                label="when the import-ap element">
+                <x:scenario
+                    label="is present">
+                    <x:expect-not-assert
+                        id="has-import-ap"
+                        label="that is correct" />
+                </x:scenario>
+            </x:scenario>
+            <x:scenario
+                label="when the import-ap element">
+                <x:scenario
+                    label="is absent">
+                    <x:context>
+                        <assessment-plan
+                            xmlns="http://csrc.nist.gov/ns/oscal/1.0">
+                            <!-- missing import-ap -->
+                        </assessment-plan>
+                    </x:context>
+                    <x:expect-assert
+                        id="has-import-ap"
+                        label="that is an error" />
+                </x:scenario>
+            </x:scenario>
+
+            <x:scenario
+                label="when the import-ap">
```

Comment:
Ditto on this an the next one - do we have a convention on when we nest scenarios?

Reply:
It is superfluous.